File: src/java/util/stream/IntStream.java
public abstract interface class java.util.stream.IntStream
{
    // Constructors
    // Methods
    public static java.util.stream.IntStream$Builder builder();
    public static java.util.stream.IntStream concat(java.util.stream.IntStream, java.util.stream.IntStream);
    public static java.util.stream.IntStream empty();
    public static java.util.stream.IntStream generate(java.util.function.IntSupplier);
    public static java.util.stream.IntStream iterate(int, java.util.function.IntUnaryOperator);
    public static java.util.stream.IntStream of(int);
    public static transient java.util.stream.IntStream of([I);
    public static java.util.stream.IntStream range(int, int);
    public static java.util.stream.IntStream rangeClosed(int, int);
    public abstract boolean allMatch(java.util.function.IntPredicate);
    public abstract boolean anyMatch(java.util.function.IntPredicate);
    public abstract java.util.stream.DoubleStream asDoubleStream();
    public abstract java.util.stream.LongStream asLongStream();
    public abstract java.util.OptionalDouble average();
    public abstract java.util.stream.Stream boxed();
    public abstract java.lang.Object collect(java.util.function.Supplier, java.util.function.ObjIntConsumer, java.util.function.BiConsumer);
    public abstract long count();
    public abstract java.util.stream.IntStream distinct();
    public abstract java.util.stream.IntStream filter(java.util.function.IntPredicate);
    public abstract java.util.OptionalInt findAny();
    public abstract java.util.OptionalInt findFirst();
    public abstract java.util.stream.IntStream flatMap(java.util.function.IntFunction);
    public abstract void forEach(java.util.function.IntConsumer);
    public abstract void forEachOrdered(java.util.function.IntConsumer);
    public volatile java.util.Iterator iterator();
    public abstract java.util.PrimitiveIterator$OfInt iterator();
    public abstract java.util.stream.IntStream limit(long);
    public abstract java.util.stream.IntStream map(java.util.function.IntUnaryOperator);
    public abstract java.util.stream.DoubleStream mapToDouble(java.util.function.IntToDoubleFunction);
    public abstract java.util.stream.LongStream mapToLong(java.util.function.IntToLongFunction);
    public abstract java.util.stream.Stream mapToObj(java.util.function.IntFunction);
    public abstract java.util.OptionalInt max();
    public abstract java.util.OptionalInt min();
    public abstract boolean noneMatch(java.util.function.IntPredicate);
    public volatile java.util.stream.BaseStream parallel();
    public abstract java.util.stream.IntStream parallel();
    public abstract java.util.stream.IntStream peek(java.util.function.IntConsumer);
    public abstract int reduce(int, java.util.function.IntBinaryOperator);
    public abstract java.util.OptionalInt reduce(java.util.function.IntBinaryOperator);
    public volatile java.util.stream.BaseStream sequential();
    public abstract java.util.stream.IntStream sequential();
    public abstract java.util.stream.IntStream skip(long);
    public abstract java.util.stream.IntStream sorted();
    public abstract java.util.Spliterator$OfInt spliterator();
    public volatile java.util.Spliterator spliterator();
    public abstract int sum();
    public abstract java.util.IntSummaryStatistics summaryStatistics();
    public abstract [I toArray();
    // Fields
}

File: src/java/util/stream/IntStream.ts
class java.util.stream.IntStream {
// Methods
public static builder(): IntStream$Builder {
return Packages.java.util.stream.IntStream.builder()
}
public static concat(int: IntStream, int_1: IntStream): IntStream {
return Packages.java.util.stream.IntStream.concat(int, int_1)
}
public static empty(): IntStream {
return Packages.java.util.stream.IntStream.empty()
}
public static generate(int: IntSupplier): IntStream {
return Packages.java.util.stream.IntStream.generate(int)
}
public static iterate(int: number, int_1: IntUnaryOperator): IntStream {
return Packages.java.util.stream.IntStream.iterate(int, int_1)
}
public static of(int: number): IntStream {
return Packages.java.util.stream.IntStream.of(int)
}
public static transient of(i: number[]): IntStream {
return Packages.java.util.stream.IntStream.of(i)
}
public static range(int: number, int_1: number): IntStream {
return Packages.java.util.stream.IntStream.range(int, int_1)
}
public static rangeClosed(int: number, int_1: number): IntStream {
return Packages.java.util.stream.IntStream.rangeClosed(int, int_1)
}
public abstract allMatch(int: IntPredicate): boolean {
return this.allMatch(int)
}
public abstract anyMatch(int: IntPredicate): boolean {
return this.anyMatch(int)
}
public abstract asDoubleStream(): DoubleStream {
return this.asDoubleStream()
}
public abstract asLongStream(): LongStream {
return this.asLongStream()
}
public abstract average(): OptionalDouble {
return this.average()
}
public abstract boxed(): Stream {
return this.boxed()
}
public abstract collect(sup: Supplier, obj: ObjIntConsumer, bic: BiConsumer): Object {
return this.collect(sup, obj, bic)
}
public abstract count(): number {
return this.count()
}
public abstract distinct(): IntStream {
return this.distinct()
}
public abstract filter(int: IntPredicate): IntStream {
return this.filter(int)
}
public abstract findAny(): OptionalInt {
return this.findAny()
}
public abstract findFirst(): OptionalInt {
return this.findFirst()
}
public abstract flatMap(int: IntFunction): IntStream {
return this.flatMap(int)
}
public abstract forEach(int: IntConsumer): void {
this.forEach(int)
}
public abstract forEachOrdered(int: IntConsumer): void {
this.forEachOrdered(int)
}
public volatile iterator(): Iterator {
return this.iterator()
}
public abstract iterator(): PrimitiveIterator$OfInt {
return this.iterator()
}
public abstract limit(lon: number): IntStream {
return this.limit(lon)
}
public abstract map(int: IntUnaryOperator): IntStream {
return this.map(int)
}
public abstract mapToDouble(int: IntToDoubleFunction): DoubleStream {
return this.mapToDouble(int)
}
public abstract mapToLong(int: IntToLongFunction): LongStream {
return this.mapToLong(int)
}
public abstract mapToObj(int: IntFunction): Stream {
return this.mapToObj(int)
}
public abstract max(): OptionalInt {
return this.max()
}
public abstract min(): OptionalInt {
return this.min()
}
public abstract noneMatch(int: IntPredicate): boolean {
return this.noneMatch(int)
}
public volatile parallel(): BaseStream {
return this.parallel()
}
public abstract parallel(): IntStream {
return this.parallel()
}
public abstract peek(int: IntConsumer): IntStream {
return this.peek(int)
}
public abstract reduce(int: number, int_1: IntBinaryOperator): number {
return this.reduce(int, int_1)
}
public abstract reduce(int: IntBinaryOperator): OptionalInt {
return this.reduce(int)
}
public volatile sequential(): BaseStream {
return this.sequential()
}
public abstract sequential(): IntStream {
return this.sequential()
}
public abstract skip(lon: number): IntStream {
return this.skip(lon)
}
public abstract sorted(): IntStream {
return this.sorted()
}
public abstract spliterator(): Spliterator$OfInt {
return this.spliterator()
}
public volatile spliterator(): Spliterator {
return this.spliterator()
}
public abstract sum(): number {
return this.sum()
}
public abstract summaryStatistics(): IntSummaryStatistics {
return this.summaryStatistics()
}
public abstract toArray(): number[] {
return this.toArray()
}
// Fields
}